src/renate/benchmark/scenarios.py

```diff
@@ -378,3 +379,27 @@ def _get_scores(self, dataset: Dataset) -> List[float]:
             )
             scores.append(value[np.argmax(count)])
         return scores
+
+
+class WildTimeScenario(Scenario):
```

Comment:
In the paper there are at least two scenarios and we may want to create a couple of variants as well.
I think the name of this class is too generic at the moment.

I also don't see if the data is split in chunks as in the paper, but I guess that's part of the package we are taking a dependency on. Is that correct?

Reply:
I think we can change the name as soon as we have a second class so we know what a good name would be.

The data is already split by time, nothing to do here.